web/themes/new_weather_theme/templates/partials/observation-table.html.twig

```diff
@@ -0,0 +1,76 @@
+<table class="usa-table usa-table--borderless observation-table border-top-1px border-base-light width-full">
+    <tbody>
+        {# WIND #}
+        <tr class="wx-wind-speed">
+            <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                {{ "Wind" | t }}
+            </th>
+            <td>
+                {% include '@new_weather_theme/partials/wind.html.twig' with { wind: content.wind } %}
+            </td>
+        </tr>
+        {# GUSTS #}
+        {%  if content.gusts is not same as (null) %}
+            <tr class="wx-gusts">
+                <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                    {{ "Gusts" | t }}
+                </th>
+                <td>
+                    {{ content.gusts.speed }} {{  "mph" | t }}
+                </td>
+            </tr>
+        {% endif %}
+        {# HUMIDITY #}
+        <tr class="wx-humidity">
+            <th scope="row" class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                {{ "Humidity" | t }}
+            </th>
+            <td>
+                {% if content.humidity == 0 %}
+                    {{  "N/A" | t}}
```

Comment:
For the the "N/A" values that appear here could you wrap them in a `<span class="text-base font-mono-xs"> ` to make consistent with the style of the N/A for wind. That would be for: 

- humidity 
- dewpoint, 
- visibility, 
- pressure

Reply:
done in https://github.com/weather-gov/weather.gov/pull/1285/commits/12281022d4a3b21167568782ba47e359ee8f1fa1